shadowsocks-csharp/Controller/Service/Sip003Plugin.cs

```diff
@@ -86,7 +87,7 @@ public bool StartIfNeeded()
                     return false;
                 }
 
-                var localPort = GetNextFreeTcpPort();
+                var localPort = int.Parse(_pluginProcess.StartInfo.Environment["SS_REMOTE_PORT"]);
```

Comment:
`SS_LOCAL_HOST` `SS_LOCAL_PORT`: the plugin listens SS native connection.
`SS_REMOTE_HOST` `SS_REMOTE_PORT`: the plugin then connect to the remote server.

SS ===> to plugin's LOCAL HOST/PORT ===> plug-in data processing ===> send to REMOTE HOST/PORT